Compilation fails on android 5

**Description**

Older android targets do not compile anymore since the introduction of the android-properties dependency.

```
android-properties/src/android/mod.rs:68: error: undefined reference to '__system_property_read_callback'
```

**Repro steps**

I'm not sure how to make an android 5 build, I'll look in to it.

**Extra materials**

Link to the failing build on Firefox's CI: https://treeherder.mozilla.org/logviewer?job_id=382181918&repo=try&lineNumber=58241

**Platform**

Android 5

--- a/wgpu-hal/Cargo.toml
+++ b/wgpu-hal/Cargo.toml
@@ -91,7 +91,7 @@ web-sys = { version = "0.3", features = ["Window", "HtmlCanvasElement", "WebGl2R
 js-sys = { version = "0.3" }
 
 [target.'cfg(target_os = "android")'.dependencies]
-android-properties = "0.2"
+android_system_properties = "0.1.1"
 
 [dependencies.naga]
 git = "https://github.com/gfx-rs/naga"

--- a/wgpu-hal/src/vulkan/instance.rs
+++ b/wgpu-hal/src/vulkan/instance.rs
@@ -573,9 +573,9 @@ impl crate::Instance<super::Api> for super::Instance {
 
         #[cfg(target_os = "android")]
         let android_sdk_version = {
+            let properties = android_system_properties::AndroidSystemProperties::new();
             // See: https://developer.android.com/reference/android/os/Build.VERSION_CODES
-            let mut prop = android_properties::getprop("ro.build.version.sdk");
-            if let Some(val) = prop.value() {
+            if let Some(val) = properties.get("ro.build.version.sdk") {
                 match val.parse::<u32>() {
                     Ok(sdk_ver) => sdk_ver,
                     Err(err) => {

